target/linux/ar71xx/image/generic.mk

```diff
@@ -196,6 +209,45 @@ define Device/mr16
 endef
 TARGET_DEVICES += mr16
 
+define Device/mr1750
+  DEVICE_TITLE := OpenMesh MR1750
+  DEVICE_PACKAGES := om-watchdog kmod-ath10k ath10k-firmware-qca988x
+  BOARDNAME = MR1750
+  FILESYSTEMS := squashfs
+  BLOCKSIZE := 4 8 16 64 128 256
```

Comment:
One blocksize and with ```k``` as unit. The blocksize should match the erase block size of the flash chip